submissions/selveresta/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="style.css" />
+		<title>Project</title>
+	</head>
+	<body>
+		<header class="header">
+			<nav class="header_nav">
+				<ul class="nav-position">
+					<li class="nav-item">
+						<a href="#">Mail</a>
+					</li>
+					<li class="nav-item">
+						<a href="#">Images</a>
+					</li>
+
+					<li class="nav-item">
+						<input type="checkbox" class="check-input" id="check" />
+						<label for="check" class="label-position">
+							<span class="box-popup"></span>
+						</label>
+						<div class="popup">
```

Comment:
Also, fix mobile view, it's the latest Chrome? but on Firefox you have an issue with kottans logo.
![image](https://user-images.githubusercontent.com/16196199/182342369-b3ce12b2-3fc2-4c33-85e3-10ae5f1dcb76.png)